packages/app/src/Element/Note.tsx

```diff
@@ -65,6 +66,12 @@ export default function Note(props: NoteProps) {
   const { data, related, highlight, options: opt, ["data-ev"]: parsedEvent, ignoreModeration = false } = props;
   const [showReactions, setShowReactions] = useState(false);
   const ev = useMemo(() => parsedEvent ?? new NEvent(data), [data]);
+  if (
+    ev.Kind === EventKind.TextNote &&
+    ev.Tags.some((a, i) => a.Key === "e" && a.Marker === "mention" && ev.Content === `#[${i}]`)
+  ) {
+    return <NoteReaction data={data} key={data.id} />;
+  }
```

Comment:
really needs?

Reply:
Don't think we need this, only place Note will be rendered outside of timeline is threads and I think is fine to render as a regular note with link to original there. We do the #[idx] substitution for content.

Reply:
thanks, removed them in https://github.com/v0l/snort/commits/d911dc8da321409787efc0a7cff6530d37894ba1.